doc/nrf/device_guides/working_with_nrf/nrf70/nrf7002eb_54l15pdk_gs.rst

```diff
@@ -0,0 +1,105 @@
+
+Overview
+********
+
+The nRF7002 EB is a shield that can be used with the nRF54L15 PDK to add support for Wi-Fi.
+
+Limitations
+===========
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode
+* It only allows Wi-Fi open security
+* It is only suitable for low throughput applications
+
+Pin mapping for nRF54L15PDK
+###########################
+
+The pin mapping for the nRF54L15 PDK is as follows:
+
++------------------+-----------------------+-----------------------------------------------+
+| nRF700x pin name | nRF54L15 PDK pins      |  Function                                     |
++==================+=======================+===============================================+
+| CLK              | P1.11                 | SPI Clock                                     |
++------------------+-----------------------+-----------------------------------------------+
+| CS               | P1.08                 | SPI Chip Select                               |
++------------------+-----------------------+-----------------------------------------------+
+| MOSI             | P1.10                 | SPI MOSI                                      |
++------------------+-----------------------+-----------------------------------------------+
+| MISO             | P1.09                 | SPI MISO                                      |
++------------------+-----------------------+-----------------------------------------------+
+| BUCKEN           | P1.13                 | Enable Buck regulator                         |
++------------------+-----------------------+-----------------------------------------------+
+| IOVDDEN          | P1.13                 | Enable IOVDD regulator                        |
++------------------+-----------------------+-----------------------------------------------+
+| IRQ              | P1.14                 | Interrupt from nRF7002                        |
++------------------+-----------------------+-----------------------------------------------+
+| GRANT            | P1.03                 | Coexistence grant from nRF7002                |
++------------------+-----------------------+-----------------------------------------------+
+| REQ              | P1.04                 | Coexistence request to nRF7002                |
++------------------+-----------------------+-----------------------------------------------+
+| STATUS           | P1.05                 | Coexistence status from nRF7002               |
++------------------+-----------------------+-----------------------------------------------+
+
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54L15 PDK
+* nRF7002 EB
+* A suitable battery
+* Micro-USB 2.0 cable
+* USB-C charger
+* Jumper wires
```

Comment:
How many jumper wires? - That should be described here

Reply:
Same as above.